features/mogelijkOnjuist.feature

```diff
@@ -281,6 +281,13 @@ Functionaliteit: Mogelijk onjuist
         """
       En bevat het antwoord geen property "mogelijkOnjuist.huisnummer"
 
+   Scenario: Openbare ruimte ligt in is in onderzoek, maar relatie naar woonplaats wordt gelegd via nummeraanduiding
```

Comment:
ligt in tussen quotes zetten, net als in de gegeven stap? Dat leest volgens mij beter